thirdparty/github.go

```diff
@@ -722,3 +722,11 @@ func buildPatchURL(gp *patch.GithubPatch) string {
 
 	return url.String()
 }
+
+func GithubPRMerge(title string, message string, sha string, mergeMethod string) error {
+	return nil
+}
+
+func GithubPostStatus(state string, description string, context string) error {
```

Comment:
Don't call this arg "context", since we also use the context package from the standard library, and the names may clash.

Reply:
Done.